back-office/shop-parameters/product-settings/product-settings-page.md

```diff
@@ -0,0 +1,146 @@
+
+Only numbers can be typed in the field, or added by using the arrows at the end of the field. It cannot be set to less than 0, otherwise the field goes back to the previous valid value and an error notification is displayed: _The Display remaining quantities when the quantity is lower than field is invalid._
+
+
+_As a merchant, I want to be able to manage my unavailable product combinations._
+
+**Display unavailable attributes on the product page.** By default, it is enabled. And the tooltip should be _If an attribute is not available in every product combination, it will not be displayed._, cf. _1.7.8_ issue #[17447](https://github.com/PrestaShop/PrestaShop/issues/17447).
+
+**When this option is enabled, unavailable products' combinations remain visible in the shop**. A message on the FO product page indicates the product is no longer available in the chosen option and invites customers to choose another combination: _Product available with different options_. 
+
+**When this option is enabled as well as the 'Allow ordering out-of-stock products' option**, customers are able to add unavailable combinations to their cart.
+ 
+**When this option is disabled and if the products have one single attribute**, their combinations are no longer available and the attribute is hidden.
+ 
+For example, the XS attribute and the related product are not displayed in the following case:
+
+| Size  |  In stock |
+|---|---|
+| XS  | No  |
+| S  | Yes  |
+| M  | Yes  |
+| L  | Yes  |
+
+but the S, M, and L attributes remain displayed.
+ 
+Also, **when this option is disabled and if the value of an attribute has no product in all possible combinations**, the combination of the products that are no longer available is hidden.
+ 
+For example, the white attribute and the related product are not displayed in the following case:
+
+| Size  | Color | In Stock |
+|---|---|---|
+| S  | White  | No |
+| S | Black | Yes  |
+| M | White | No |
+| M | Black | Yes |
+
+but the S, M, and black attributes remain displayed.
+
+
+_As a merchant, I want to be able to parameter my products' attributes._
+
+**Display the "Add to cart" button when a product has attributes.** By default, it is enabled. Display or hide the 'Add to cart' button on category pages for products that have attributes forcing customers to see product details.
+
+**Separator of attribute anchor on the product links.** When a product has many attributes, the merchant can **select a separator between the hyphen (-) and the comma (,)**. By default, it is the hyphen. It appears on shopping carts in the front office, invoices, merchandise returns, and order details pages, cf. issue #[19636](https://github.com/PrestaShop/PrestaShop/issues/19636).
```

Comment:
If I understand well the issue, the seperator will not be used only in the product links
So, maybe we can change the label of this toggle ?

Reply:
It is mainly a product setting even if it can be found in other parts of the store so I think the label must stay connected to the product. That being said, I could report it in the front office specs as well.